docs/setup-aws-lambda.asciidoc

```diff
@@ -0,0 +1,88 @@
+[[aws-lambda]]
+=== Get started with Lambda (Experimental)
+
+experimental::[]
+
+Getting Elastic APM set up for your lambda functions is easy,
+and there are various ways you can tweak it to fit your needs.
+
+NOTE: In order to get the full AWS Lambda tracing capabilities, use with APM Server 7.16 or higher. Using
+older versions provides most value, but some relevant metadata fields will not be indexed.
+
+[float]
+[[aws-lambda-installation]]
+==== Installation
+
+Setting up your Lambda function is a two-step process.
+
+1. <<aws-lambda-extension>>
+2. <<aws-lambda-instrumenting>>
+
+[float]
+[[aws-lambda-extension]]
+==== Install the Elastic APM Lambda extension
+
+Elastic uses a Lambda extension to forward data to APM Server in a way that does not interfere with the
+execution of your Lambda function.
+
+See the https://github.com/elastic/apm-aws-lambda/blob/main/docs/aws-lambda-extension.asciidoc[installation documentation] to get started.
+
+NOTE: Not all environment variables specified within https://github.com/elastic/apm-aws-lambda/blob/main/docs/aws-lambda-extension.asciidoc#the-necessary-variables[The Necessary Variables]
+section of the extension documentation need to be configured. The Java agent utilizes a https://docs.aws.amazon.com/lambda/latest/dg/runtimes-modify.html[wrapper script]
+to set some of these up, so follow the next section to see which are required.
+
+[float]
+[[aws-lambda-instrumenting]]
+==== Installing the Elastic APM Java Agent layer
+
+The Java Agent is installed as an AWS Layer as well. It is distributed as a ZIP archive containing two files:
+
+- The Java Agent jar file
+- A wrapper script that sets up the runtime to enable automatic attachment of the agent
+
+Follow these steps to ensure proper Java Agent attachment to your Lambda function:
+
+1. Download the `elastic-apm-java-aws-lambda-layer-<VERSION>.zip` archive from our
+https://github.com/elastic/apm-agent-java/releases[GitHub Releases page].
+2. Publish the downloaded ZIP archive as an https://docs.aws.amazon.com/lambda/latest/dg/invocation-layers.html?icmpid=docs_lambda_help[AWS Lambda Layer].
+3. Configure your function to use the Java Agent layer.
+4. Within your Function configuration, configure the following environment variables:
+a. `AWS_LAMBDA_EXEC_WRAPPER` (required): set this variable's value to `/opt/elastic-apm-handler`, so that Lambda starts the runtime with this script.
+b. `ELASTIC_APM_LAMBDA_APM_SERVER` (required): this will tell the Elastic APM Extension where to send data to.
+c. `ELASTIC_APM_SECRET_TOKEN` or `ELASTIC_APM_API_KEY` (required): see the relevant https://github.com/elastic/apm-aws-lambda/blob/main/docs/aws-lambda-extension.asciidoc#the-necessary-variables[documentation].
```

Comment:
Minor thing: I think this cross link here is not required. Just copy the text from there as it is quite short (would improve the experience):

This controls the authentication method that extension uses when sending data to the URL configured via `ELASTIC_APM_LAMBDA_APM_SERVER`.

Reply:
Done 👍